interfaces/builtin/modem_manager.go

```diff
@@ -266,13 +266,73 @@ const modemManagerPermanentSlotUDev = `
 # Concatenation of all ModemManager udev rules
 # do not edit this file, it will be overwritten on update
 
-ACTION!="add|change|move", GOTO="mm_cinterion_port_types_end"
-SUBSYSTEM!="tty", GOTO="mm_cinterion_port_types_end"
-ENV{ID_VENDOR_ID}!="1e2d", GOTO="mm_cinterion_port_types_end"
+ACTION!="add|change|move|bind", GOTO="mm_cinterion_port_types_end"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="1e2d", GOTO="mm_cinterion_port_types"
+GOTO="mm_cinterion_port_types_end"
 
+LABEL="mm_cinterion_port_types"
 SUBSYSTEMS=="usb", ATTRS{bInterfaceNumber}=="?*", ENV{.MM_USBIFNUM}="$attr{bInterfaceNumber}"
 
-ATTRS{idVendor}=="1e2d", ATTRS{idProduct}=="0053", ENV{.MM_USBIFNUM}=="01", ENV{ID_MM_CINTERION_PORT_TYPE_GPS}="1"
+# PHS8
+ATTRS{idVendor}=="1e2d", ATTRS{idProduct}=="0053", ENV{.MM_USBIFNUM}=="01", SUBSYSTEM=="tty", ENV{ID_MM_PORT_TYPE_GPS}="1"
```

Comment:
Since we enter the block starting with the `mm_cinterion_port_types` label only on the condition `ATTRS{idVendor}=="1e2d"`, then I guess that the same condition could be omitted from all the lines in the block? Or is it still good udev practice to keep it?

Also, I was a bit confused that the match on the subsystem inside the block is on `tty`, whereas in the line with the GOTO condition is `usb`; but then I noticed that in the latter it's `SUBSYSTEMS` with the plural, so I guess this matches on a list? Just asking to make sure that this is not a typo.

Reply:
1. I think you are correct that the conditions Ould be omitted, however it is a good practice to keep the lines as they are not so that it is easy to read. The combined file can be long and include rules for many devices and so it is easier to have all the condition in one line rather than to track which section it is.

2. There is no typo, although it might look like so. In this case both SUBSYSTEMS and ATTRS conditions apply to the parent usb_device (idVendor attribute is only available in the usb_device), so they apply to all ports of the same device.

Reply:
Thanks Konrad for the explanation, LGTM! :-)